src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/CacheHealthDecider.java

```diff
@@ -69,8 +70,8 @@ public CacheHealthDecider(
             .put(ResourceEnum.FIELD_DATA_CACHE, fieldDataCacheClusterRca)
             .build();
 
-    this.fieldDataCacheSizeUpperBound = CacheDeciderConfig.DEFAULT_FIELD_DATA_CACHE_UPPER_BOUND;
-    this.shardRequestCacheSizeUpperBound = CacheDeciderConfig.DEFAULT_SHARD_REQUEST_CACHE_UPPER_BOUND;
+    this.fieldDataCacheSizeUpperBound = getDefaultShardRequestCacheUpperBound();
+    this.shardRequestCacheSizeUpperBound = getDefaultFieldDataCacheUpperBound();
```

Comment:
Add a comment on why we're always going with the default here instead of allowing overrides.

Reply:
These fields also make certain calls in this class super misleading. e.g. line 167 on this file has a call to getFieldDataCacheUpperBound(), but that calls Decider#getFieldDataCacheUpperBound which reads the value from a configObj and not this field right? What's the purpose of these fields?

Reply:
I am not sure on the purpose of these fields. I have removed these fields as the the default value would be added in case the config is not read from the file.